src/components/Logo/Logo.tsx

```diff
@@ -15,16 +14,14 @@ const Logo: React.FC = () => {
     logo = darkMode ? indexLogoFullWhite : indexLogoFullBlack
   else logo = darkMode ? indexLogoWhite : indexLogoBlack
 
-  console.log(window.innerWidth)
```

Comment:
lol same idea: https://github.com/SetProtocol/index-ui/pull/504

^ I'll close my PR in favor of this